File: MusicManagerAPI/src/main/java/com/ldz/music/manager/model/bo/MusicTypeBO.java
package com.ldz.music.manager.model.bo;

import com.ldz.music.manager.constants.MusicTypes;

/**
 * Created by ldalzotto on 26/04/2017.
 */
public class MusicTypeBO {

    private MusicTypes musicTypes;

    private String sourceUrl;

    public MusicTypes getMusicTypes() {
        return musicTypes;
    }

    public void setMusicTypes(MusicTypes musicTypes) {
        this.musicTypes = musicTypes;
    }

    public String getSourceUrl() {
        return sourceUrl;
    }

    public void setSourceUrl(String sourceUrl) {
        this.sourceUrl = sourceUrl;
    }
}

File: MusicManagerAPI/src/scala/com/ldz/music/manager/model/bo/MusicTypeBO.scala
package com.ldz.music.manager.model.bo

import com.ldz.music.manager.constants.MusicTypes

import scala.beans.BeanProperty

/**
  * Created by Loic on 20/06/2017.
  */
case class MusicTypeBO(@BeanProperty musicTypes: MusicTypes.Value, @BeanProperty sourceUrl: String) {

}
